sound/soc/intel/common/soc-acpi-intel-cml-match.c

```diff
@@ -182,8 +182,8 @@ static const struct snd_soc_acpi_link_adr cml_3_in_1_mono_amp[] = {
 	},
 	{
 		.mask = BIT(1),
-		.num_adr = ARRAY_SIZE(rt1308_1_adr),
-		.adr_d = rt1308_1_adr,
+		.num_adr = ARRAY_SIZE(rt1308_1_single_adr),
+		.adr_d = rt1308_1__single_adr,
```

Comment:
``rt1308_1__single_adr`` should be ``rt1308_1_single_adr``

Reply:
yes, I thought I fixed this but obviously didn't. Thanks for catching this.